package.json

```diff
@@ -67,13 +70,17 @@
     "@types/semver": "^5.3.30",
     "chai": "^3.5.0",
     "github": "^8.2.1",
+    "husky": "^0.14.3",
     "json-stringify-pretty-compact": "^1.0.3",
     "mocha": "^3.2.0",
     "npm-run-all": "^4.0.2",
     "nyc": "^10.2.0",
+    "prettier": "^1.13.7",
```

Comment:
Like the documentation pointed out https://prettier.io/docs/en/install.html
Use exact version instead of ^ or ~
Only use 1.13.7

Reply:
Right. I changed it.